### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by svkrish**
I lead the development for Mobile Foundation on ICPA.  I request the proposed changes to be able to provide better visibility for Mobile Foundation's capabilities in the Cloud Pak.

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -124,3 +124,7 @@ After the installation completes, links to your {{site.data.keyword.icpa_short}}
 1. Click **Service Catalog**, then select **Application Console**.  You should see several projects installed.
    - If you click the **kabanero** project, the Overview tab contains the URLs to the **icpa-landing** page and the **tekton-dashboard**.
    - If you click the **ta** project, the Overview tab contains the URL to the {{site.data.keyword.ta_short}} UI.
+   
+## Step 7. Install IBM Mobile Foundation (optional)
+{: step7}
+Optionally, if you wish extend the out-reach of your modernised and cloud-native apps via multi-channel digital apps to front-end install IBM Mobile Foundation.  See [here](http://mobilefirstplatform.ibmcloud.com/tutorials/en/foundation/8.0/ibmcloud/deploy-mf-on-ibmcloud-ocp/)for a quick install.
```
Comment (jantley-ibm) [inline getting-started.md]: Change:
Optionally, if you wish extend the out-reach of your modernised and cloud-native apps via multi-channel digital apps to front-end install IBM Mobile Foundation.  See [here](http://mobilefirstplatform.ibmcloud.com/tutorials/en/foundation/8.0/ibmcloud/deploy-mf-on-ibmcloud-ocp/)for a quick install.
to
Optionally, if you want to extend the outreach of your modernized and cloud-native apps via multi-channel digital apps to front-end, install IBM Mobile Foundation.  See [Deploy Mobile Foundation on Red Hat OpenShift Container Platform on IBM Cloud](http://mobilefirstplatform.ibmcloud.com/tutorials/en/foundation/8.0/ibmcloud/deploy-mf-on-ibmcloud-ocp/) for installation instructions.

File: getting-started.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ subcollection: cloud-pak-applications
...
+{{site.data.keyword.icpa_rm}} is an enterprise-ready, containerized software solution for modernizing existing applications and developing new cloud-native apps that run on Red Hat&reg; OpenShift&reg;. This hybrid, multicloud foundation breaks down technology and data silos to make modernization faster and more secure, and speeds development of applications built for Kubernetes. You can access cloud services and meet the technology standards and policies your company requires.  Also,  with IBM Mobile Foundation in this Cloud Pak you can build and deploy the next generation of multi channel digital apps, including mobile, wearables, conversation, web, and PWAs that extend the out-reach of your modernized and cloud native apps.
```
[the remark this answers is not in the document]
Comment (svkrish) [reply getting-started.md]: Am good with this proposed change

File: getting-started.md (hunk 1 above)
Comment (svkrish) [reply getting-started.md]: Am good with this proposed change

**Comment by helyarp** [comment]
@svkrish Hi, reviews for ICPA doc on IBM Cloud are handled by @jantley-ibm

**Comment by jantley-ibm** [comment]
@svkrish Since you submitted the PR, make the changes and push them to the same branch and they will be reflected in the existing PR.  Once I review and approve then I will merge.

**Comment by svkrish** [comment]
@jantley-ibm thanks.   I have made the changes to the same branch.   Please review and merge.

**Comment by jantley-ibm** [comment]
@svkrish I have learned that it is not possible to merge your changes, this master stream is a mirror of the production stream.  I have made your changes manually using our normal process.  Please check and close this PR if you're satisfied with the changes.

**Comment by svkrish** [comment]
@jantley-ibm the changes look good... thanks very much.... am closing this PR.

**Pull request closed without merging**